l10n_th_base_sequence/__manifest__.py

```diff
@@ -1,13 +1,14 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
 
 {
-    "name": "Base Sequence",
+    "name": "Thailand - Base Sequence",
```

Comment:
```suggestion
    "name": "Thai Localization - Base Sequence",
```
I think you should use **Thai Localization** instead. What do you think?